### ISSUE TITLE
The theory of multi-label  & CrossEntropyLoss

### ISSUE THREAD

**Issue by HJiaoBME** [miraclewkf/multilabel-MXNet#5]
Dear @miraclewkf,
Thanks for your code.  I am doing multi-label classification based on deep learning, but I have encountered some problems. I can't find much theoretical information.  I can't quite understand the cross entropy loss of multi-label classifications and the back-transfer of gradients. Can you recommend related papers?  I will combine the code and read it carefully and analyze it. Thank you very much.
